src/main/kotlin/com/example/cms/domains/preparedcontents/services/beans/PreparedContentServiceBean.kt

```diff
@@ -66,7 +66,7 @@ class PreparedContentServiceBean @Autowired constructor(
         this.validate(entity)
         val placeholders = entity.placeholderValues.mapValues {
             if (it.key == DYNAMIC_CONTENT_KEY) {
-                HtmlTable.fromJson(it.value, listOf("dynamic-table"))
+                HtmlTable.fromJson(it.value, entity.cssClasses?.split(" ") ?: listOf("dynamic-table"))
```

Comment:
`entity.cssClasses?.split(" ") ?: listOf("dynamic-table")`

this field is non-null, so what's the point of `?: listOf("dynamic-table")`